apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/AIPolicyHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import { IS_PLATFORM } from 'lib/constants'
+import { AiIcon, Button } from 'ui'
+
+export const AIPolicyHeader = ({
+  assistantVisible,
+  setAssistantVisible,
+}: {
+  assistantVisible: boolean
+  setAssistantVisible: (v: boolean) => void
+}) => {
+  return (
+    <div className="space-y-1 py-4 px-4 bg-overlay sm:px-6 border-b border-overlay">
+      <div className="flex flex-row justify-between items-center">
+        <span className="text-sm">Create a new row level security policy</span>
```

Comment:
muscle memory wanted a tab autocomplete to happen here
https://share.cleanshot.com/d67TyGZn